### ISSUE TITLE
import_page encoding problem

### ISSUE THREAD

**Issue by davidak** [getnikola/plugins#141]
```
$ nikola import_page http://en.wikipedia.org/wiki/Information_extraction
[2016-02-29T20:39:33Z] INFO: requests.packages.urllib3.connectionpool: Starting new HTTP connection (1): en.wikipedia.org
[2016-02-29T20:39:34Z] INFO: requests.packages.urllib3.connectionpool: Starting new HTTPS connection (1): en.wikipedia.org
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/doit/doit_cmd.py", line 168, in run
    return command.parse_execute(args)
  File "/usr/local/lib/python3.5/site-packages/doit/cmd_base.py", line 122, in parse_execute
    return self.execute(params, args)
  File "/usr/local/lib/python3.5/site-packages/nikola/plugin_categories.py", line 124, in execute
    return self._execute(options, args)
  File "/Users/davidak/code/Webseite/plugins/import_page/import_page.py", line 61, in _execute
    self._import_page(url)
  File "/Users/davidak/code/Webseite/plugins/import_page/import_page.py", line 68, in _import_page
    title = doc.find('*//title').text_content().decode('utf-8')
AttributeError: 'lxml.etree._ElementUnicodeResult' object has no attribute 'decode'
```

Python 3.5.1
Nikola vv7.7.6

@ralsina

**Comment by ralsina** [comment]
Works on py2 so it's "one of those". I'll take a look tonight.

**Comment by ralsina** [comment]
Doesn't look very hard. For example, this works:

```python
import pip
pip.main(['freeze'])
```

**Comment by ralsina** [comment]
I just tried it with python 3.4 and it worked for me :-P

**Comment by evanfoster** [comment]
I can reproduce this issue.

```bash
nikola import_page http://en.wikipedia.org/wiki/Information_extraction
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/doit/doit_cmd.py", line 168, in run
    return command.parse_execute(args)
  File "/usr/lib/python3.5/site-packages/doit/cmd_base.py", line 122, in parse_execute
    return self.execute(params, args)
  File "/usr/lib/python3.5/site-packages/nikola/plugin_categories.py", line 129, in execute
    return self._execute(options, args)
  File "/tmp/foo/plugins/import_page/import_page.py", line 66, in _execute
    self._import_page(url)
  File "/tmp/foo/plugins/import_page/import_page.py", line 73, in _import_page
    title = doc.find('*//title').text_content().decode('utf-8')
AttributeError: 'lxml.etree._ElementUnicodeResult' object has no attribute 'decode'
```

Environment: Arch Linux
Python: 3.5.2
Nikola: 7.8.1

Please let me know if you need me to run any commands or need more information.

**Comment by Kwpolska** [comment]
Reproduced. I’ll look into it.

**Comment by Kwpolska** [comment]
Fixed in commit be697bd — please run `nikola plugin -i import_page` again.
